src/main/java/pl/confitura/jelatyna/demo/DemoSender.java

```diff
@@ -0,0 +1,24 @@
+package pl.confitura.jelatyna.demo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import pl.confitura.jelatyna.email.EmailParams;
+import pl.confitura.jelatyna.email.Sender;
+
+@Service
+@Profile("demo")
+public class DemoSender implements Sender {
```

Comment:
imho it should have name StubSender (or FakeSender) and be in pakage pl.confitura.jelatyna.email.stub (or fake)

and profile name should be "stub" or "dev"